#diff -r 4572f6055a6a toolkit/locales/en-US/toolkit/about/aboutSupport.ftl
#--- a/toolkit/locales/en-US/toolkit/about/aboutSupport.ftl	Tue Mar 26 11:51:06 2019 +0200
#+++ b/toolkit/locales/en-US/toolkit/about/aboutSupport.ftl	Tue Mar 26 17:24:45 2019 +0100
#@@ -62,7 +62,7 @@
#         [windows] Open Folder
#        *[other] Open Directory
#     }
#-modified-key-prefs-title = Important Modified Preferences
#+modified-key-prefs-title = All Modified Preferences
# modified-prefs-name = Name
# modified-prefs-value = Value
# user-js-title = user.js Preferences
diff -r b1146cce5053 toolkit/locales/en-US/toolkit/about/aboutSupport.ftl
--- a/toolkit/locales/en-US/toolkit/about/aboutSupport.ftl	Tue Jun 23 02:23:48 2020 +0300
+++ b/toolkit/locales/en-US/toolkit/about/aboutSupport.ftl	Tue Jun 23 07:37:02 2020 +0200
@@ -78,7 +78,7 @@
 experimental-features-title = Experimental Features
 experimental-features-name = Name
 experimental-features-value = Value
-modified-key-prefs-title = Important Modified Preferences
+modified-key-prefs-title = All Modified Preferences
 modified-prefs-name = Name
 modified-prefs-value = Value
 user-js-title = user.js Preferences
#diff -r 4572f6055a6a toolkit/modules/Troubleshoot.jsm
#--- a/toolkit/modules/Troubleshoot.jsm	Tue Mar 26 11:51:06 2019 +0200
#+++ b/toolkit/modules/Troubleshoot.jsm	Tue Mar 26 17:24:45 2019 +0100
#@@ -17,6 +17,7 @@
# // entries are *prefixes*: for example, "accessibility." applies to all prefs
# // under the "accessibility.*" branch.
# const PREFS_WHITELIST = [
#+  "",
#   "accessibility.",
#   "apz.",
#   "browser.cache.",
#@@ -119,7 +120,7 @@
# 
#   return PREFS_WHITELIST.reduce(function(prefs, branch) {
#     Services.prefs.getChildList(branch).forEach(function(name) {
#-      if (filter(name) && !PREFS_BLACKLIST.some(re => re.test(name)))
#+      if (filter(name) )//&& !PREFS_BLACKLIST.some(re => re.test(name))) //commented out to return all prefs! so about:support will list all prefs not just the important ones!
#         prefs[name] = getPref(name);
#     });
#     return prefs;
diff -r e704e3565db9 toolkit/modules/Troubleshoot.jsm
--- a/toolkit/modules/Troubleshoot.jsm	Thu Jul 11 00:49:58 2019 +0300
+++ b/toolkit/modules/Troubleshoot.jsm	Thu Jul 11 03:48:15 2019 +0200
@@ -21,6 +21,7 @@
 // entries are *prefixes*: for example, "accessibility." applies to all prefs
 // under the "accessibility.*" branch.
 const PREFS_WHITELIST = [
+  "",
   "accessibility.",
   "apz.",
   "browser.cache.",
@@ -128,7 +129,7 @@
 
   return PREFS_WHITELIST.reduce(function(prefs, branch) {
     Services.prefs.getChildList(branch).forEach(function(name) {
-      if (filter(name) && !PREFS_BLACKLIST.some(re => re.test(name))) {
+      if (filter(name) ) {//&& !PREFS_BLACKLIST.some(re => re.test(name))) { //commented out to return all prefs! so about:support will list all prefs not just the important ones!
         prefs[name] = getPref(name);
       }
     });
